wnbrb/wnb-rb-site
# Past meetups page

This PR creates the past meetups/archive page (Issue https://github.com/wnbrb/wnb-rb-site/issues/9)! Big thanks to @ayreal for helping me with React and styling 🙏 I'm about to be out of town so edits are welcome, otherwise I'll get back to any feedback in the new year.

Some notable changes:
- Updated the API to return past meetups as a nested hash year->month->meetups and include speaker info
- Added an additional meetup to the seeds so we could make sure the styling of the timeline looked right

Steps to test:
- Reseed with `bundle exec rake db:seed`
- Navigate to `/meetups`
- See past meetups with speaker info, ordered from latest first

![image](https://user-images.githubusercontent.com/12875392/146261168-537521b9-b2f7-4090-85a5-0ae75faa74b3.png)
